clientconn.go

```diff
@@ -496,15 +499,16 @@ type ClientConn struct {
 // connections accordingly.  If doneChan is not nil, it is closed after the
 // first successfull connection is made.
 func (cc *ClientConn) lbWatcher(doneChan chan struct{}) {
+	isPickFirst := reflect.TypeOf(cc.dopts.balancer) == reflect.TypeOf(&pickFirst{})
 	for addrs := range cc.dopts.balancer.Notify() {
 		var (
 			add []Address   // Addresses need to setup connections.
 			del []*addrConn // Connections need to tear down.
 		)
 		cc.mu.Lock()
-		for _, a := range addrs {
-			if _, ok := cc.conns[a]; !ok {
-				add = append(add, a)
+		for _, addr := range addrs {
```

Comment:
This change is gratuitous and go style prefers shorter variable names for small loops like this (as it was before); please revert.